starter_code/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Relaxr</title>
+  </head>
+  <body>
+  <img src="/Users/jeauxy/Code/WDI/homework/relaxr_html/starter_code/images/header_bg.jpg"/>
```

Comment:
The src should be the relative path to the file from this current file, not the relative path on your computer. i.e. `<img src="images/header_bg.jpg"/>`